backend/services/mock.js

```diff
@@ -51,7 +51,7 @@ async function createMockData() {
     permissions: {
```

Comment:
Can we update these permissions as well, President should have all of the permissions